From: Stefano Rivera <stefanor@debian.org>
Date: Tue, 24 Dec 2024 12:25:30 -0400
Subject: Install doc sources

It's nice to provide RST as well as HTML docs.

Forwarded: not-needed
---
 docs/conf.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/conf.py b/docs/conf.py
index fb80943..8aa81e8 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -13,7 +13,7 @@ html_css_files = [
 ]
 #html_theme = "shibuya"
 
-html_copy_source = False
+html_copy_source = True
 html_show_sourcelink = False
 
 language = "en"
